client/api/request.js

```diff
@@ -49,6 +49,15 @@ const _request = ( url, data, method, namespace = '' ) => {
 				return;
 			}
 
+			if ( 403 === response.status ) {
+				const message = __(
+					'The request was not fulfilled. ' +
+					'Please see the status page for diagnostics: ' +
+					'WooCommerce » Status » WooCommerce Services.'
```

Comment:
Can we make this more friendly? Is it possible to link this to the status page?

Reply:
Ping @dechov - let's change this to "The request was not fulfilled. Please see the [status page](/wp-admin/admin.php?page=wc-status&tab=connect) for diagnostics." (with the correct URL)